MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -1002,15 +1015,18 @@ private void BuildStreamVideoItem(
                 // Reverse codec profiles for backward compatibility - first codec profile has higher priority
                 .Reverse();
 
-            foreach (var codecProfile in appliedAudioConditions)
+            if (audioCodec is not null)
```

Comment:
I think this check will become unnecessary when https://github.com/jellyfin/jellyfin/pull/12521 is backported.